backend/apollo/elections/views.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import generics
+
+from apollo.elections.models import Question, Election
+from apollo.elections.serializers import QuestionSerializer, ElectionSerializer
+
+
+class QuestionDetail(generics.RetrieveUpdateDestroyAPIView):
+    queryset = Question.objects.all()
+    serializer_class = QuestionSerializer
+
+
+class ElectionDetail(generics.RetrieveUpdateDestroyAPIView):
+    queryset = Election.objects.all()
+    serializer_class = ElectionSerializer
+
+
+class ElectionList(generics.ListCreateAPIView):
+    queryset = Election.objects.all()
+    serializer_class = ElectionSerializer
```

Comment:
I'd go for a viewset instead of multiple views (and generally if think it's a good idea to default to a ViewSet).

An equivalent code using a viewset would be
```python
class ElectionViewSet(viewsets.ModelViewSet):
    queryset = Election.objects.all()
    serializer_class = ElectionSerializer
```
and in urls we'd need to use a router (a simple example is in the docs https://www.django-rest-framework.org/tutorial/6-viewsets-and-routers/). For now it's not really necessary, but in most cases viewsets scale much better than views.